app/models/sac_cas/concerns/role_beitragskategorie.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2023, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+require_relative '../../../domain/sac_cas/beitragskategorie/calculator'
+
+module ::SacCas::RoleBeitragskategorie 
+  extend ActiveSupport::Concern
+
+  included do
+    include I18nEnums
+    i18n_enum :beitragskategorie, %w(einzel jugend familie).freeze
```

Comment:
Fände ich schöner wenn dieses Array von der Calculator Klasse käme, dann ist die definitive Auflistung aller möglichen Stati in einer einzigen Datei drin. Sonst müssen wir beim Hinzufügen von neuen Stati auch an diese Datei hier denken.